akka-remote/src/main/resources/reference.conf

```diff
@@ -689,8 +722,19 @@ akka {
         parallelism-max = 2
       }
     }
-
-
   }
+}
+
+# Artery low latency configuration profile.
+# To enable this profile, add the following to your settings:
+#   akka.remote.artery.advanced = ${artery-profile-low-latency}
+artery-profile-low-latency {
+  idle-cpu-level = 9
+}
```

Comment:
if we can't find any other properties that should be derived from  "profile" I don't think we should do it